### DISCUSSION TITLE
Can I create a listing with 100% of the blog content, like in a stream?

### DISCUSSION THREAD

**Discussion by bryanwhiting** [quarto-dev/quarto-cli#8039] [Q&A]
I can choose from table, grid, list, etc. 

But I'd like one view that just shows the content right there. Just an endless stream of blog posts, similar to https://seths.blog/).

Is there a way to write a [custom listing](https://quarto.org/docs/websites/website-listings.html#listing-item-fields) to enable this? Seems like there isn't a listing field to pull "content" from the post.

**Comment by mcanouil** [comment]
I don't think you can.
But you can use `description` to put all the content.

**Reply by bryanwhiting** [reply]
Thanks! I assume there are limits of type of content (images, markdown styling like block quotes) that go into the description?

**Reply by mcanouil** [reply]
Maybe, you'll have to try for yourself ...

**Comment by bryanwhiting** [comment]
Ahh! You've got me. Indeed, this exercise is left for the reader.

Thanks! And happy holidays.

- Bryan
________________________________
